app/src/services/gfw.service.ts

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+
+import logger from 'logger';
+
+class GfwService {
+    static async getFields(urlDataset: string): Promise<Record<string, any>[]> {
+        logger.debug(`Obtaining fields of ${urlDataset}`);
+        const reqUrl: string = `${urlDataset}/fields`;
+        try {
+            const result: Record<string, any> = await axios.get(reqUrl);
+            return result.data;
+        } catch (err) {
+            logger.error('Error obtaining fields', err);
+            throw new Error('Error obtaining fields');
+        }
+    }
+
+    static async executeQuery(
```

Comment:
this method and the following can probably be DRYed up

Reply:
Done.. not sure why it's not labeled as outdated.